template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Areas/IdentityService/Data/SeedClientApplication.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Company.WebApplication1.Identity.Data;
+using JetBrains.Annotations;
+
+namespace Company.WebApplication1.Identity.Data.Migrations
+{
+    public static class SeedClientApplication
+    { 
+        public static void Seed(MigrationBuilder migrationBuilder)
+        {
+            var applicationColumns = new string[] { "Id", "ClientId", "Name" };
+            var appId = Guid.NewGuid().ToString();
```

Comment:
Is there any way to get the templating engine to inline a new Guid? It seems bad that you get a new one every time you apply this migration.